src/main/java/uk/gov/ons/ctp/response/notify/config/NotifyConfiguration.java

```diff
@@ -20,6 +20,11 @@
   private String apiKey;
   private String debugHttpCode;
 
+  // If addressOverride is true, what address should the mails be sent to?
+  private String overrideAddress;
+  // Should all the emails be sent to a single email address?
+  private Boolean addressOverride;
```

Comment:
Could do with a comment that the override address to be sent to has to be a whitelisted team member and that this is for testing.